theurgy.lic

```diff
@@ -625,7 +689,7 @@ class TheurgyActions
 
   def meditate_bead
     return false unless get_prayer_chain?
-    if count_prayer_beads.zero?
+    if count_prayer_beads == 0
```

Comment:
why the change from if count_prayer_beads.zero?

Reply:
Reverted.